### PR TITLE
Expand FSRS section 

### PR DESCRIPTION
**Pull request by cqg2**
Text mosty taken from the program's info boxes and from the FSRS tutorial on github, with permission of its author:

https://github.com/open-spaced-repetition/fsrs4anki/blob/main/docs/tutorial.md

Added a new option in "Deck options"

"Weights" changed for "Paramaters"
https://github.com/ankitects/anki/commit/329ac92c5e42cc9cc6fc8ab4d452551684200926

### CODE REVIEW

File: src/deck-options.md [hunk 1]
```diff
@@ -411,12 +414,126 @@ to forget, it can help you remember more material in the same amount of time.
...
 
+
+
+- **Maximum interval.**
+This works in the same way as the default algorithm: it is the maximum number
+of days a review card will wait. For more info, please see [next section](#maximum-interval)
+
+- **Desired Retention.**
+The most important setting to configure is the desired retention:this will
+determine the fraction of cards recalled successfully when they are due.
```
Comment (dae) [inline src/deck-options.md:433]: Adding the graph from that page you linked that shows red bars on the left and right would help users to visualise what's going on here. 

I suggest rewriting to the following:

*Desired retention* controls how likely you are to remember cards when they are reviewed. The default value of 0.9 will schedule cards so you have a 90% chance of remembering them when they come up for review again.

Here is a graph that shows how adjusting this value will affect your workload:

[insert graph]

There are two things to notice:

- As desired retention approaches 1.0, the frequency that you need to review cards increases drastically. For example, imagine you have a card that you have a 90% chance of remembering after 100 days. If your desired retention was 0.95, you'd need to review it after 47 days instead (approximately twice as frequently). At 0.97, the delay would be only 27 days (approximately 3.7x as frequently). At 0.99, you'd be reviewing every 9 days (more than 10x what you'd be doing with the defaults).
- As desired retention decreases, you'll forget a greater percentage of your cards, and those cards will need to be reviewed again. Eventually, you'll get to a point where the forgotten cards contribute more to your workload than you gain from the longer delays, which is why you see the workload on the left of the graph increasing. Also, bear in mind that forgetting material frequently is demotivating.

For these reasons, we suggest you be conservative when adjusting this number, and recommend you keep it between 0.85 and 0.95.

File: src/deck-options.md [hunk 2]
```diff
@@ -411,12 +414,126 @@ to forget, it can help you remember more material in the same amount of time.
...
+When FSRS is enabled, the learning and re-learning steps should be chosen 
+in such a way that all the learning steps can be completed on the same day. 
+For most users, this means that steps longer than or equal to 1 day should 
+not be used. The reason is that FSRS can determine more optimal intervals but
+the use of longer (re)learning steps doesn't allow FSRS to schedule the reviews, 
+making the scheduling less optimal. In addition, if longer steps are used, 
+there can be cases where the "Hard" interval exceeds the "Good" interval.
+
+The use of multiple short (re)learning steps is also not recommended when using FSRS.
+
```
[the remark this answers is not in the document]
Comment (Expertium) [reply src/deck-options.md:526]: > The hard/good thing is not an issue in Anki's implementation.

I actually didn't know that, thank you

File: src/deck-options.md [hunk 3]
```diff
@@ -411,12 +414,126 @@ to forget, it can help you remember more material in the same amount of time.
...
+when FSRS is enabled.
 
 If you previously used the 'custom scheduling' version of FSRS, please make
 sure you clear out the custom scheduling section before enabling FSRS.
 
+
+
+- **Maximum interval.**
+This works in the same way as the default algorithm: it is the maximum number
+of days a review card will wait. For more info, please see [next section](#maximum-interval)
```
[the remark this answers is not in the document]
Comment (cqg2) [reply src/deck-options.md:429]: IMO, some users may find confusing that "maximum interval" settings appearns on the "Advanced" section (when using SM2) and then, when activating FSRS disappears from there to appear again under FSRS option. I personally, not being familiar with FSRS had to got to the wiki to learn that it's actually exactly the same setting, it wasn't inmediatly obvious to me, that's why I left that reference. 

I've removed the redundant "Maximum interval" here, but as a suggestion for future versions, I'd suggest to take FSRS subesction out of the "Advanced" section of the deck options.

File: src/deck-options.md [hunk 4]
```diff
@@ -411,12 +414,126 @@ to forget, it can help you remember more material in the same amount of time.
...
+
+   Once you've done 1000+ reviews in Anki, you can use the Optimize button to 
+analyze your review history, and automatically generate parameters that are
+optimal for your memory and the content you're studying. Parameters are 
+preset-specific, so if you have decks that vary wildly in difficulty, it
+is recommended to assign them separate presets, as the parameters for easy
+decks and hard decks will be different. There is no need to optimize your
+parameters frequently - once every few months is sufficient.
+
+   By default, parameters will be calculated from the review history of all
```
[the remark this answers is not in the document]
Comment (cqg2) [reply src/deck-options.md:481]: Not sure what you mean here, sorry; as far as I know, the indentation is consistent with the rest of the document.

File: src/deck-options.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cqg2) [reply src/deck-options.md:536]: added, thanks

File: src/deck-options.md (hunk 1 above)
Comment (cqg2) [reply src/deck-options.md]: added, thanks!

File: src/deck-options.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (dae) [reply src/deck-options.md:489]: Most of the manual doesn't indent the start of a paragraph.

**Comment by dae** [comment]
Thanks @cqg2!

**Pull request merged**